File: Archer Codify 1/Gauri Awate/Java/Java Practices/BubbleSort.java
import java.util.*;
class BubbleSort
{
	public static void main(String args[])
	{
		Scanner sc=new Scanner(System.in);
		System.out.println("\nEnter the count:");
		int cnt=sc.nextInt();
		int x[]=new int[cnt];
		int i,j,tmp;
		System.out.println("\n Enter any 5 nos: ");
	for(i=0;i<x.length;i++)
	{
		x[i]=sc.nextInt();
	}

	System.out.println("\n Before Sort: ");
	for(i=0;i<x.length;i++)
	{
		System.out.println(" "+x[i]);
	}

	
	for(i=0;i<cnt-1;i++)
	{
		for(j=0;j<(cnt-1-i);j++)
		{
			if(x[j]>x[j+1])
			{
				tmp=x[j];
				x[j]=x[j+1];
				x[j+1]=tmp;
				
			}
		}
	}
	System.out.println("\n After Sort: ");
	for(i=0;i<x.length;i++)
	{
		System.out.println(" "+x[i]);
	}

	}
}

File: Archer Codify 1/Aditya Kamble/C/Bubble_Sort.c
#include<stdio.h>
#define SIZE 5

int main()
{
    int arr[SIZE];
    int i,j;
    int tmp;

    printf("\n Enter any 5 no:");
    for(i=0; i<5; i++)
    {
        scanf("%d", &arr[i]);
    }

    for(i=0; i<5; i++)
    {
        for(j=0; j<(4-i); j++)
        {
            if(arr[j]>arr[j+1])
            {
                tmp=arr[j];
                arr[j]=arr[j+1];
                arr[j+1]=tmp;
            }
        }
    }
    printf("\n After Sort: ");
    for(i=0; i<5; i++)
    {
        printf("%5d",arr[i]);
    }

    return 0;

}